index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './src';
```

Comment:
We probably want to export more stuff. E.g. at least the interfaces for `Transport` and such, in case we want to replace the `LocalTransport` with another form of `Transport`.